packages/gatsby-theme-aio/src/components/GetCredential/Return/ReturnProduct.js

```diff
@@ -15,18 +15,22 @@ const ReturnProduct = ({ productList }) => {
                 }
               `}
             >
-              <TooltipTrigger delay={0}>
-                <ActionButton aria-label="Edit Name">
-                  <img
-                    src={product?.icon}
-                    css={css`
-                      width: 35px;
-                      cursor : pointer;
-                    `}
-                  />
-                </ActionButton>
-                <Tooltip>{product?.label}</Tooltip>
-              </TooltipTrigger>
+              {product?.icon ? (
+                <TooltipTrigger delay={0}>
+                  <ActionButton aria-label="Edit Name">
+                    <img
+                      src={product?.icon}
+                      css={css`
+                        width: 35px;
+                        cursor: pointer;
+                      `}
+                    />
+                  </ActionButton>
+                  <Tooltip>{product?.label}</Tooltip>
+                </TooltipTrigger>
+              ) : (
+                <p className="spectrum-Body spectrum-Body--sizeS">{product?.label}</p>
+              )}

```

Comment:
I see duplicate code in Products.js. Can we merge ReturnProduct and Products.js?